rapyuta_io_sdk_v2/utils.py

```diff
@@ -85,3 +86,51 @@ def get_default_app_dir(app_name: str) -> str:
     # On Linux and other Unix-like systems
     xdg_config_home = os.environ.get("XDG_CONFIG_HOME", os.path.expanduser("~/.config"))
     return os.path.join(xdg_config_home, app_name)
+
+
+# Decorator to handle server errors and munchify response
+# def handle_and_munchify_response(func):
+#     def wrapper(*args, **kwargs):
+#         response = func(*args, **kwargs)
+#         handle_server_errors(response)
+#         return munchify(response.json())
+
+#     return wrapper
+
+
+def handle_and_munchify_response(func):
+    def wrapper(*args, **kwargs):
+        # Call the synchronous function
+        response = func(*args, **kwargs)
+        handle_server_errors(response)
+        return munchify(response.json())
+
+    async def async_wrapper(*args, **kwargs):
+        # Call the function asynchronously
+        response = await asyncify(func)(*args, **kwargs)
+        # handle_server_errors(response)
+        # return munchify(response.json())
+        return response
```

Comment:
todo: Update this based on what we have decided to do.

Reply:
I will revert it back to previous code but when async client will be made this code will be helpful as we have to await for response in sync wrapper